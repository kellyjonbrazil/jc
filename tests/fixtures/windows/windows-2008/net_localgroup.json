{
  "account_origin": "\\\\WIN-SERVER8",
  "domain": null,
  "comment": null,
  "groups": [
    {
      "name": "Administrators",
      "members": []
    },
    {
      "name": "Backup Operators",
      "members": []
    },
    {
      "name": "Certificate Service DCOM Access",
      "members": []
    },
    {
      "name": "Cryptographic Operators",
      "members": []
    },
    {
      "name": "Distributed COM Users",
      "members": []
    },
    {
      "name": "Event Log Readers",
      "members": []
    },
    {
      "name": "Guests",
      "members": []
    },
    {
      "name": "IIS_IUSRS",
      "members": []
    },
    {
      "name": "Network Configuration Operators",
      "members": []
    },
    {
      "name": "Performance Log Users",
      "members": []
    },
    {
      "name": "Performance Monitor Users",
      "members": []
    },
    {
      "name": "Power Users",
      "members": []
    },
    {
      "name": "Print Operators",
      "members": []
    },
    {
      "name": "Remote Desktop Users",
      "members": []
    },
    {
      "name": "Replicator",
      "members": []
    },
    {
      "name": "Users",
      "members": []
    }
  ]
}
