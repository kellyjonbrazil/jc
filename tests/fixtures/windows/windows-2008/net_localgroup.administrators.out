Alias name     Administrators
Comment        Administrators have complete and unrestricted access to the computer/domain

Members

-------------------------------------------------------------------------------
Administrator
user1
vagrant
The command completed successfully.